_data/bootloaders.json

```
@@ -9,6 +9,12 @@
         "esp32s2": {
             "version": "0.9.0"
         },
+        "esp32s3": {
+            "version": "0.9.0"
+        },
+        "esp32c3": {
+            "version": "0.9.0"
+        },
```

```suggestion
        "esp32c3": {},
```
Since there is no build of tinyuf2 for C3, we should leave it empty.

---

Good call.